authors/apps/articles/tests/base_test.py

```diff
@@ -13,10 +13,16 @@ def setUp(self):
         # urls
         self.register_url = reverse("authentication:user-signup")
         self.login_url = reverse("authentication:user-login")
+        self.register_url = reverse("authentication:user-signup")
```

Comment:
Duplicates

Reply:
I have resolved this but this  is not getting outdated